app/domain/authentication/authn_jwt/orchestrate_jwt_authentication.rb

```diff
@@ -16,13 +16,18 @@ module AuthnJwt
       )
 
       def call
-        authenticate
+        authenticate_jwt
       end
 
       private
 
-      def authenticate
-        @token_factory.signed_token(
+      def get_vendor
```

Comment:
this should be `vendor` or `vendor_name`. Ruby methods do not have `get` in there name.